build-all.bash

```diff
@@ -1,23 +1,15 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
+set -vex
 
-# This script builds all Docker containers described by Dockerfiles in this repository
-# For each Dockerfile it runs: 
-# $ docker build -t $TAGPREFIX/$TAG 
-# and
-# $ docker push -t $TAGPREFIX/$TAG
-#
-# $TAGPREFIX is defined in this file. 
-# If a Dockerfile has a Dockertag file in the same directory, it's read from there
-# Otherwise $TAG is generated by replacing / and <space> with - in the directory name. 
-#
-# docker build command uses --no-cache option to prevent stale layers to be used. 
+HERE="$( cd "$(dirname "$0")" ; pwd -P )"
+
+export RELEASE_HASH="${RELEASE_HASH:-$(curl -s https://api.github.com/repos/iterative/dvc/releases/latest | sha256sum | cut -c -8)}"
+export TAG_PREFIX="${TAG_PREFIX:-dvcorg}"
 
-set -vex
 
-RELEASE_HASH=$(curl -s https://api.github.com/repos/iterative/dvc/releases/latest | sha256sum | cut -c -8)
+./generate-all.bash
```

Comment:
Why create a separate `generate-all.bash` yet not use it anywhere else? Either keep things in one file or re-use them in multiple places, surely?

Reply:
Removed this call to `generate-all.bash`. For debugging and security, `generate`, `build`, `push` should be called separately IMO.